src/pudl/metadata/codes.py

```diff
@@ -231,6 +231,21 @@
         },
         "ignored_codes": [],
     },
+    "boiler_generator_association_types_eia": {
+        "df": pd.DataFrame(
+            columns=[
+                "code",
+                "label",
+                "description",
+            ],
+            data=[
+                ("A", "actual", "An actual boiler generator association."),
+                ("T", "theorecial", "A theoretical boiler generator association."),
```

Comment:
Can we provide any additional color on what actual/theoretical BGA means here? Does EIA clarify this at all?